### PR TITLE
Added hungarian messages

### PR DESCRIPTION
**Pull request by montlikadani**

### CODE REVIEW

File: resources/config.yml [hunk 1]
```diff
@@ -53,4 +54,4 @@ gun-item: iron_barding
 #Enable halloween special pinata effects?
 halloween-mode: false
 
-File-Version-Do-Not-Edit: 5
+File-Version-Do-Not-Edit: 6
```
Comment (Plajer) [inline resources/config.yml]: No need to change file version adding only comment.

File: src/pl/plajer/pinata/Main.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
     private static Main instance;
     private final int MESSAGES_FILE_VERSION = 10;
-    private final int CONFIG_FILE_VERSION = 5;
+    private final int CONFIG_FILE_VERSION = 6;
```
Comment (Plajer) [inline src/pl/plajer/pinata/Main.java]: No need changing it.

**Review by Plajer — CHANGES_REQUESTED** [review]
Do not modify file version and it will be ok.

Note for future: only English, Polish and German locales will be available in the future.

File: src/pl/plajer/pinata/Main.java (hunk 2 above)
Comment (montlikadani) [reply src/pl/plajer/pinata/Main.java]: Ooops.. I'm sorry, I did not know.

**Review by Plajer — APPROVED** [review]

File: src/pl/plajer/pinata/Main.java (hunk 2 above)
Comment (Plajer) [reply src/pl/plajer/pinata/Main.java]: I should say: "No need to change it if you added only comment to the config.yml" :)

**Pull request merged**